hab-pkg-deb/plan.sh

```diff
@@ -0,0 +1,49 @@
+pkg_name=hab-pkg-deb
+pkg_origin=chef
+pkg_version=0.1.0
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('Apache-2.0')
+pkg_source=nosuchfile.tar.gz
+# dpkg-deb depends on GNU tar, which is installed as a dependency of core/hab.
+pkg_deps=(
+  core/bash
+  core/coreutils
+  core/grep
+  core/sed
+  core/util-linux
+  core/dpkg
+  core/findutils
+  core/hab
+  core/node
```

Comment:
Do we depend directly on node or is this a transitive dependency of handlebars-cmd? If it's the latter we can remove it.